iothub/service/src/Feedback/MessageFeedbackProcessorClient.cs

```diff
@@ -116,11 +115,6 @@ public virtual async Task OpenAsync(CancellationToken cancellationToken = defaul
                     throw new Exception("Callback for message feedback must be set before opening the connection.");
                 }
 
-                if (_amqpConnection.IsOpen())
```

Comment:
Moving this check into the AmqpConnectionHandler instead since all 3 subclients call into it and all 3 subclients need this check